### DISCUSSION TITLE
Revisions don't work together with translations

### DISCUSSION THREAD

**Discussion by ikovac** [directus/directus#22886] [Feature Requests]
### Describe the Bug

Revisions do not function with the translations field. I understand that this is a relational field and that revisions are not intended to work with relations. However, translations are a hidden collection, and I believe that in this case, we should display changes and have the ability to revert changes accurately, rather than just showing an "update at" change. 
Newly created translation collections are not intended to be edited on their own, which is why they are hidden. Therefore, it might be worth considering showing those changes when navigating through revisions.

What are your opinions? Have you already discussed this topic in the past, and what are the conclusions?

Preview:
![Kapture 2024-07-01 at 10 32 29](https://github.com/directus/directus/assets/26693037/48a4b594-70b5-49ab-8c1d-9a43e8b15643)

### To Reproduce

1. Create a languages collection
2. Add a few languages
3. Create articles collection
4. Add translations field and link it to languages collection
5. Add title field to the translations field
6. Enable content versioning for the article collection
7. Create a new article
8. Update title and save the article
9. Click on revisions - the only changed field is "updated at"

### Directus Version

v10.12.1

### Hosting Strategy

Self-Hosted (Custom)

### Database

postgres latest

**Comment by rijkvanzanten** [comment]
Heya! All revisions are tracked on the individual items that were changed, so the revision themselves should still be visible on the detail page of the actual translation item that was changed. That being said, I totally agree showing them inline in the parent item that changed the nested items would be a good enhancement. I will move this to a feature request though, as it needs a bit more thinking through and designing before we can go over to implementation
